index.html

```diff
@@ -408,6 +404,9 @@
         <h2>
           Sequence diagram
         </h2>
+        <p class="issue">
+          This needs updating and there is no source in the repo (naughty).
```

Comment:
I've added the sequence diagram source in vdx format. I made it in Lucidchart, and editing it should work well there.